lib/rubocop/cop/rspec/mixin/top_level_group.rb

```diff
@@ -28,10 +28,6 @@ def on_top_level_example_group(_node); end
 
         def on_top_level_group(_node); end
 
-        def top_level_group?(node)
-          top_level_groups.include?(node)
-        end
-
```

Comment:
Note that while we may not be using this method, it is a mixin and may be used by custom Cops that rely on the RuboCop RSpec. And since we are using semantic versioning, we do not recommend removing this method until the next major version update. I would suggest just printing a deprecation warning first.

Reply:
I found 3 users of this on Github:

https://github.com/search?q=%22top_level_group%3F%22&type=code

I will look into how to test this. 

I'm not even sure we should remove it from the public API?

Also if you have a quick example of why this needs to be defined so that a subclass can overwrite it (is that even what's going on?), it would definitely save me some time. The examples I saw don't call `super` so I feel like I'm missing something.

Reply:
I split these changes to:

* https://github.com/rubocop/rubocop-rspec/pull/1976

I will drop the `top_level_group?` change from this PR and update the description to match.